fe/src/main/java/org/apache/doris/analysis/ExportStmt.java

```diff
@@ -221,10 +229,17 @@ private static void checkPath(String path) throws AnalysisException {
         throw new AnalysisException("Invalid export path. please use valid 'HDFS://', 'AFS://' or 'BOS://' path.");
     }
 
-    private void checkProperties(Map<String, String> properties) throws AnalysisException {
+    private void checkProperties(Map<String, String> properties) throws UserException {
         this.columnSeparator = PropertyAnalyzer.analyzeColumnSeparator(
                 properties, ExportStmt.DEFAULT_COLUMN_SEPARATOR);
         this.lineDelimiter = PropertyAnalyzer.analyzeLineDelimiter(properties, ExportStmt.DEFAULT_LINE_DELIMITER);
+        if (properties != null && properties.containsKey(LoadStmt.EXEC_MEM_LIMIT)) {
+            try {
+                this.execMemLimit = Long.parseLong(properties.get(LoadStmt.EXEC_MEM_LIMIT));
+            } catch (NumberFormatException e) {
+                throw new DdlException("Execute memory limit is not Long", e);
+            }
+        }
```

Comment:
if properties not contain this property, you can get it from SessionVariable.

Reply:
OK